lib/miq_expression/target.rb

```diff
@@ -64,4 +64,19 @@ def target
       reflections.last.klass
     end
   end
+
+  def tag_path_with(value = nil)
+    # encode embedded / characters in values since / is used as a tag seperator
+    "#{tag_path}#{value.nil? ? '' : '/' + value.to_s.gsub(/\//, "%2f")}"
```

Comment:
encoding:
What do you mean? To use some ruby method for enconding?
there is problem only with '\/'